x-pack/plugins/security_solution/public/common/components/event_details/overview/overview_card.tsx

```diff
@@ -106,25 +82,27 @@ export const OverviewCardWithActions: React.FC<OverviewCardWithActionsProps> = (
   contextId,
   dataTestSubj,
   enrichedFieldInfo,
-}) => {
-  const [isPopoverVisisble, setIsPopoverVisible] = useState(false);
-
-  return (
-    <OverviewCard title={title} isPopoverVisible={isPopoverVisisble}>
-      <EuiFlexGroup alignItems="center" gutterSize="none">
-        <ClampedContent data-test-subj={dataTestSubj}>{children}</ClampedContent>
-
-        <ActionWrapper>
-          <ActionCell
-            {...enrichedFieldInfo}
-            contextId={contextId}
-            setIsPopoverVisible={setIsPopoverVisible}
-            applyWidthAndPadding={false}
-          />
-        </ActionWrapper>
-      </EuiFlexGroup>
-    </OverviewCard>
-  );
-};
+}) => (
+  <OverviewCard title={title}>
+    <EuiFlexGroup alignItems="center" gutterSize="none">
+      <ClampedContent data-test-subj={dataTestSubj}>{children}</ClampedContent>
+
+      <ActionWrapper>
+        <CellActions
+          field={{
+            name: enrichedFieldInfo.data.field,
+            value: enrichedFieldInfo?.values ? enrichedFieldInfo?.values[0] : '',
```

Comment:
We should be able to support any kind of value here
```suggestion
            value: enrichedFieldInfo?.values,
```

Reply:
We do support it. But the field in the UI doesn't. It only displays the first value.
Here is one of the places using it: 
https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/components/event_details/overview/index.tsx#L185

If it gets more than one value on the array, CellActions would consider both values, while the UI only shows one. I am afraid of updating it because `EnrichedFieldInfo` is a complex feature to test.